src/newFunction.ts

```diff
@@ -0,0 +1,6 @@
+function newFunction() {
+    let msg = "Hello World!";
+    console.log(msg);
+}
+
+newFunction();
```

Comment:
You should be exporting the function and importing in the CLI script.

Reply:
Could you please clarify how this would be done? Where does the function get exported to and where does the CLI script get imported from?

Reply:
Export it like:

```
export {
  newFunction
};
```

At the end of a typescript module.

Then import it in your command script like `import { newFunction } from '../lib/...';`

Try this so you can practice importing and exporting code.

If you look at js-polykey this is how we structure our code.